pages/signup.js

```diff
@@ -36,10 +42,10 @@ export default function Signup() {
   });
 
   useEffect(() => {
-    if (accountType.length > 0 && signupInfo.username.length > 0 && signupInfo.firstname.length > 0 && signupInfo.lastname.length > 0 && signupInfo.email.length > 0 && signupInfo.password.length > 0  && signupInfo.location.length > 0) {
+    if (accountType.length > 0 && signupInfo.username.length > 0 && signupInfo.firstname.length > 0 && signupInfo.lastname.length > 0 && signupInfo.email.length > 0 && signupInfo.password.length > 0 && signupInfo.location.length > 0) {
```

Comment:
Just make a note to take care of this later

Reply:
LOL sorry